### PR TITLE
Make Categorization and PostOptions localizable

### PR DESCRIPTION
**Pull request by MarcioBarrientos**
Allow to make localizable Blog Categorization tab and BlogPost Options tab

### CODE REVIEW

**Comment by dhensby** [comment]
I've noticed that you've created this pull request from one of the "core branches" rather than a branch specifically intended for this pull request. This can cause problems if we ask you to target this PR at a different branch. For the time being no action is required, this is just informational for the next time you create a PR.

In the future please create a new branch for each individual pull request. See our [guide about contributing PRs](https://docs.silverstripe.org/en/4/contributing/code/#step-by-step-from-forking-to-sending-the-pull-request)

**Comment by dhensby** [comment]
Thanks for this, it looks like you might have some linting issues, please can you correct the code style problems?

File: src/Admin/GridFieldMergeAction.php [hunk 1]
```diff
@@ -101,7 +101,8 @@ public function getColumnContent($gridField, $record, $columnName)
                 'data-target' => $prefix . '-target-record-' . $record->ID
             ]);
 
-            return $dropdown->Field() . $action->Field() . '<a title="Move posts to" class="MergeActionReveal">move posts to</a>';
+            return $dropdown->Field() . $action->Field() . '<a title="' . _t(__CLASS__ . '.MovePostsTo', 'Move posts to') . '" class="MergeActionReveal">' . _t(__CLASS__ . '.MovePostsTo', 'Move posts to') . '</a>';
```
Comment (dhensby) [inline src/Admin/GridFieldMergeAction.php]: line length here is too long

**Comment by MarcioBarrientos** [comment]
Thanks for the advice, I didn't read the guide.

File: src/Admin/GridFieldMergeAction.php [hunk 2]
```diff
@@ -101,7 +101,11 @@ public function getColumnContent($gridField, $record, $columnName)
                 'data-target' => $prefix . '-target-record-' . $record->ID
             ]);
 
-            return $dropdown->Field() . $action->Field() . '<a title="Move posts to" class="MergeActionReveal">move posts to</a>';
+            $MovePostsTo = _t(__CLASS__ . '.MovePostsTo', 'Move posts to');
+            $MergeActionReveal = '<a title="' . $MovePostsTo . '" class="MergeActionReveal">' . $MovePostsTo . '</a>';
+
+            return $dropdown->Field() . $action->Field() . $MergeActionReveal;
+
```
Comment (raissanorth) [inline src/Admin/GridFieldMergeAction.php]: This blank line will have to be removed for the Travis build to pass, thanks!

File: src/Model/Blog.php [hunk 3]
```diff
@@ -178,7 +178,10 @@ public function getCMSFields()
                 ]
             );
 
-            $fields->findOrMakeTab('Root.Categorisation')->addExtraClass('blog-cms-categorisation');
+            $fields->fieldByName('Root.Categorisation')
+                ->addExtraClass('blog-cms-categorisation')
+                ->setTitle(_t(__CLASS__ . '.Categorisation', 'Categorisation'));
+
```
Comment (raissanorth) [inline src/Model/Blog.php]: The closing brace should go on the next line, please. That's for the Travis build too.

**Comment by raissanorth** [comment]
I have left a couple of comments regarding the failing Travis tests. You could run `phpcs` and `phpcbf`which are provided by the codesniffer tool included with blog as a dependency: https://github.com/silverstripe/silverstripe-blog/blob/master/composer.json#L19

**Pull request closed without merging**